expfmt/text_parse.go

```diff
@@ -60,6 +60,7 @@ type TextParser struct {
 	currentMF            *dto.MetricFamily
 	currentMetric        *dto.Metric
 	currentLabelPair     *dto.LabelPair
+	currentLabel         []*dto.LabelPair
```

Comment:
I know that protobuf uses the singular even for repeated messages. But maybe it would still make more sense and be more readable to use the plural here? `currentLabels` collides with the map of the same name below, but maybe call this `currentLabelPairs`? Also, is there a good comment to add here to explain what this variable is for? (cf. the comment for the `currentLabels` variable below).